packages/app-project/src/screens/ProjectAboutPage/components/AboutDropdownNav.spec.js

```diff
@@ -0,0 +1,21 @@
+import { shallow} from 'enzyme'
+import { AboutDropdownNav } from './AboutDropdownNav'
+
+describe('Component > AboutDropdownNav', () => {
+  let wrapper
+
+  const router = {
+    query: {
+      owner: 'test-owner',
+      project: 'test-project'
+    }
+  }
+
+  before(() => {
+    wrapper = shallow(<AboutDropdownNav aboutNavLinks={[]} router={router} />)
+  })
+
+  it('should render without crashing', () => {
+    expect(wrapper).to.be.ok()
+  })
```

Comment:
Could we expand the tests here to test for the conditional rendering happening in the dropdown nav similar to what you have in the specs for `AboutSidebar`?

Reply:
@srallen Because the dropdown uses `DropButton` from `grommet`, the `AboutNavLink` components in the `dropContent` are not rendered until a user clicks the drop button. I tried a few techniques like simulating a click or using `setProps` in the spec file, but couldn't get the menu to 'open' in tests.

Anything you recommend trying with `enzyme` to get the dropdown menu to render?

Reply:
Per our convo this morning, I separated the dropdown content into its own component for testing conditional rendering of the nav links 👍